### DISCUSSION TITLE
Word count, reading time, refresh feeds progress and ability to disable logging.

### DISCUSSION THREAD

**Discussion by TheOneImprove** [jocmp/capyreader#590] [Feature Requests]
I highly recommend adding the following features:
- Word count for articles.
- Reading time for articles, displayed inside the article view.
- Refresh progress for feeds (Checkout Agr Reader on google playstore, for implementation ideas.)
- Adding the ability to disable logging in the app.

**Comment by TheOneImprove** [comment]
I also recommend having a sticky button to put the search or mark all as read on, to make the main bar cleaner.

**Comment by jocmp** [comment]
Thanks for the suggestions. I know other readers have word count and read times but I don't want to add these to the app in the interest of lower information density the reading mode and list elements.

About logging, the app doesn't log outside of exceptions which are generated by the system. These are always kept local to device except when using the Google Play release which has an opt-in to send anonymized crash data to Crashlytics.
